CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # Unreleased
+### Added
+- Set the environment variable for Azure pipeline and Cloud Build Agent machine to avoid skewing aggregated telemetry.
```

Comment:
Consider "- Collect environment variables in telemetry distinguishing Azure Pipelines and Cloud Build"

"Set the env..." could be read the wrong way that we are setting env vars.

The Changelog need not explain that deeply about why we are doing something, just the What, so that a developer can quickly see what's been done and decide for themselves if that change impacts them.